src/issue.ts

```diff
@@ -179,7 +180,12 @@ async function getRepoIssues(repo: Repo, flags?: Flags): Promise<IssueResult> {
         console.warn(
           `Error fetching issues for ${repo.repo}. Sleeping for ${sleepTime} milliseconds.`
         );
-        console.warn(e);
+        // strip keys from the url
+        const errorText = inspect(e, false, 100).replace(
+          /key=.*'/,
+          '[redacted]'
```

Comment:
nit: suggest `"key=[redacted]'"` to leave the closing apostrophe in place, and to show what exactly is redacted.

Reply:
good call out!